code/modules/screen_alert/command_alert.dm

```diff
@@ -77,9 +77,12 @@
 		return
 	for(var/mob/faction_receiver in alert_receivers)
 		if(faction_receiver.faction == human_owner.faction || isdead(faction_receiver))
-			faction_receiver.play_screen_text("<span class='maptext' style=font-size:24pt;text-align:center valign='top'><u>COMMAND ANNOUNCEMENT:</u></span><br>" + text, /atom/movable/screen/text/screen_text/command_order)
+			var/faction_title = "Command" // the default title, for tgmc announcements
+			if(human_owner.faction != FACTION_TERRAGOV)
+				faction_title = "[GLOB.faction_to_acronym[human_owner.faction]] Command" // if the sender doesn't belong to the TGMC, clarify the faction in the title
```

Comment:
Should probs not assume the faction will always be in the list (even if it should)

Reply:
how should I implement this? I am basically clueless as to how I would make it check if the leader's faction is in that list or something

Reply:
ternary or something